docs/src/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Welcome to "The Steel Book" &mdash; a guide for a dynamic and versatile programming language
+designed to harmonize with Rust, known as Steel. Born from the Lisp family and descended from
+Scheme, Steel offers a blend of dynamic expressiveness within Rust's strict environment, creating a
+powerful synergy.
+
+But why choose a Lisp dialect as an extension language?
+
+Rust provides robustness and safety, ensuring reliable and efficient software development. However,
+its rigidity can sometimes feel limiting when facing dynamic and evolving requirements, such as
+those encountered in a text editor, a crucial tool that developers tailor to their needs. Enter
+Lisp &mdash; a language family renowned for its extensibility and dynamic nature. It serves as the
+ultimate extension language, designed to be extendable itself by allowing developers to create DSLs
+and tweak the evaluation process to fit any domain or task, even those unforeseen or irrelevant to
+the original application.
+
+In "The Book of Steel," our aim is twofold: to demystify language basics for developers unfamiliar
```

Comment:
Nit - this should now be `"The Steel Book"`

Reply:
oops, thanks